### DISCUSSION TITLE
Dutch sentences that should be supported

### DISCUSSION THREAD

**Discussion by balloob** [OHF-Voice/intents#374] [Supported Sentences]
If you don't feel comfortable contributing to the YAML in this repository, you can post your **Dutch** sentences that match [our supported intents](https://github.com/home-assistant/intents/blob/main/intents.yaml) below.

**Comment by codeBaker1989** [comment]
I would do some dutch translations :)

**Reply by TheFes** [reply]
Sorry, I missed this message. 

Please have a look at the the `nl` sentences. If you think anything is missing, feel free to add them.

If you have any questions, please let me know.

**Comment by synesthesiam** [comment]
Thanks @codeBaker1989! Please coordinate with @TheFes :)

**Reply by legovaer** [reply]
@TheFes I have some availability this week. Need some help?

**Reply by TheFes** [reply]
@legovaer Right now we have translations for all intents which were selected to start with, however there are probably still improvements possible. You can have a look and see if there is anything missing which you think would be used as a voice command.
